### PR TITLE
Get llvmlite 23 from source

### PR DESCRIPTION
**Pull request by mrodozov**
Get llvmilite 23 from source, now using as it's default backend LLVM 6
https://github.com/numba/llvmlite/commit/9ae78b184965f76d32b2120c25216cabe23bb3c4
Not yet available as pip pkg

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: py2-llvmlite.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
-### RPM external py2-llvmlite 0.22.0
+
+### RPM external py2-llvmlite 0.23.0
 ## INITENV +PATH PYTHON27PATH %{i}/${PYTHON_LIB_SITE_PACKAGES}
-## INITENV +PATH PYTHON3PATH %{i}/${PYTHON3_LIB_SITE_PACKAGES}
```
Comment (smuzaffar) [inline py2-llvmlite.spec]: @mrodozov , we need to keep this line. This is needed for python3 env

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**